bestiary/core/schema.py

```diff
@@ -344,17 +345,19 @@ class Arguments:
         ecosystem_id = graphene.ID()
         name = graphene.String()
         title = graphene.String(required=False)
+        parent_project_id = graphene.ID(required=False)
```

Comment:
I think `parent_id` works better because it's shorter. You should modify it in the model too.